PhantomIntelligenceHeader header

uint16 id
PhantomIntelligencePixel[] pixels
